types/node/index.d.ts

```diff
@@ -2129,17 +2129,17 @@ declare module "url" {
         constructor(init?: URLSearchParams | string | { [key: string]: string | string[] | undefined } | Iterable<string[]>);
```

Comment:
Should we change the `Iterable<string[]>` here (and in the implements statement) also to `implements Iterable<[string, string]>`?

Reply:
Great, point, I've included that as well.